pkg/newstorage/newstorage.go

```
@@ -99,16 +109,15 @@ type Store interface {
 
 	// Batch performs multiple Put and/or Delete operations in order.
 	// Depending on the implementation, this method may be faster than repeated Put and/or Delete calls.
-	Batch(operations []Operation) error
```

You should return a generic error type from functions.